### PR TITLE
Fix heartbeat typo

### PR DESCRIPTION
**Pull request by Jarema**
Signed-off-by: Tomasz Pietrek <tomasz@nats.io>

# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by Jarema** [comment]
Hearbeats were introduced after 1.9, so this should not be a breaking change for released verison AFAIK.

**Review by piotrpio — APPROVED** [review]
LGTM

File: pubsub/jetstream/metadata.go [hunk 1]
```diff
@@ -141,8 +141,8 @@ func parseMetadata(psm pubsub.Metadata) (metadata, error) {
 		m.rateLimit = v
 	}
 
-	if v, err := time.ParseDuration(psm.Properties["hearbeat"]); err == nil {
-		m.hearbeat = v
+	if v, err := time.ParseDuration(psm.Properties["heartbeat"]); err == nil {
```
Comment (ItalyPaleAle) [inline pubsub/jetstream/metadata.go:144]: This is a breaking change however. Can you please:

1. Add a fallback logic so if the `hearbeat` property (with typo) is set, but not `heartbeat`, then `hearbeat` is used.
2. Please update the documentation for the component at https://github.com/dapr/docs too

**Review by ItalyPaleAle — CHANGES_REQUESTED** [review]

**Comment by ItalyPaleAle** [comment]
Oh nevermind, I missed the note that heartbeat were not released yet. Then LGTM :) But please, do open an issue in https://github.com/dapr/docs

**Review by ItalyPaleAle — APPROVED** [review]

**Review by berndverst — APPROVED** [review]
I was wondering in the last PR why this said `hearbeat` but then I thought maybe this was a special thing in upstream Jetstream and given you are the authority @Jarema I didn't question it further.

**Comment by berndverst** [comment]
> Tests seem to be stuck?

Not stuck - just a ton of worker requests pending. It has happened increasingly in contrib.

**Pull request merged**